bundles/org.openhab.binding.modbus.solaxx3mic/README.md

```diff
@@ -0,0 +1,296 @@
+                        registerType="SHORT",
+                        registerUnit="AMPERE",
+                        registerScaleFactor=-1
+                ]
+                Type ac-phase-frequency-type: acPhaseC#ac-phase-frequency [
+                        registerNumber=9,
+                        registerFunction=3,
+                        registerType="SHORT",
+                        registerUnit="HERTZ",
+                        registerScaleFactor=-2
+                ]
+                Type ac-phase-power-type: acPhaseC#ac-phase-power [
+                        registerNumber=18,
+                        registerFunction=3,
+                        registerType="SHORT",
+                        registerUnit="WATT",
+                        registerScaleFactor=0
+                ]
+                Type dc-current-type: dcGeneral1#dc-current [
+                        registerNumber=2,
+                        registerFunction=3,
+                        registerType="SHORT",
+                        registerUnit="AMPERE",
+                        registerScaleFactor=-1
+                ]
+                Type dc-voltage-type: dcGeneral1#dc-voltage [
+                        registerNumber=0,
+                        registerFunction=3,
+                        registerType="SHORT",
+                        registerUnit="VOLT",
+                        registerScaleFactor=-1
+                ]
+                Type dc-power-type: dcGeneral1#dc-power [
+                        registerNumber=19,
+                        registerFunction=3,
+                        registerType="SHORT",
+                        registerUnit="WATT",
+                        registerScaleFactor=0
+                ]
+                Type dc-current-type: dcGeneral2#dc-current [
+                        registerNumber=3,
+                        registerFunction=3,
+                        registerType="SHORT",
+                        registerUnit="AMPERE",
+                        registerScaleFactor=-1
+                ]
+                Type dc-voltage-type: dcGeneral2#dc-voltage [
+                        registerNumber=1,
+                        registerFunction=3,
+                        registerType="SHORT",
+                        registerUnit="VOLT",
+                        registerScaleFactor=-1
+                ]
+                Type dc-power-type: dcGeneral2#dc-power [
+                        registerNumber=21,
+                        registerFunction=3,
+                        registerType="SHORT",
+                        registerUnit="WATT",
+                        registerScaleFactor=0
+                ]
+}
+```
+
+SolaxX3Mic.items
+
+```
+Group House
+Group Sensors_Solax_X3_Mic
+Number modbus_inverter_Solax_X3_Mic_deviceInformation_cabinet_temperature "Cabinet Temperature [%.1f °C]" <temperature> (Sensors_Solax_X3_Mic) {channel="modbus:inverter-solax-x3-mic:Solax-X3-Mic:deviceInformation#cabinet-temperature" }
```

Comment:
All these items are bare Number, without dimension. Shouldn't they have dimension as well, e.g. Number:Temperature?

Reply:
I do not understand your point. How should I do that?